frontend/styles/var/mixins.scss

```diff
@@ -123,6 +123,12 @@ $max-width: 2560px;
   filter: grayscale(0.5);
   pointer-events: none;
   cursor: default;
+
+  // Must override
+  &:hover,
+  &:focus {
+    cursor: default;
+  }
```

Comment:
@iansltx I added it directly into the `disabled` mixin here so it'll be applied on anything using this mixin